lib/cocoapods/target/pod_target.rb

```diff
@@ -66,6 +66,10 @@ class PodTarget < Target
     #
     attr_reader :app_spec_build_settings
 
+    # @return [Hash{String => (String,PodTarget)}]
+    #
+    attr_accessor :test_app_hosts_by_spec_name
```

Comment:
definitely open for a better name for this method

Reply:
not particularly happy with mutating the `pod_target` after it has been created. Can this be calculated beforehand and passed in?

Reply:
this is no different than the (releated) dependent_target-type methods, and indeed needs to be done after initialization